sdv/metadata/table.py

```diff
@@ -205,6 +205,51 @@ def _update_transformer_templates(self, rounding, min_value, max_value):
                 'float': custom_float
             })
 
+    @staticmethod
+    def _get_constraint_sort_key(constraint):
+        try:
+            if constraint.handling_strategy == 'reject_sampling':
+                return 0
+            else:
+                return 1
+        except AttributeError:
+            return 1
+
+    def _sort_constraints(self):
+        for idx, constraint in enumerate(self._constraints):
+            if isinstance(constraint, type):
+                constraint = constraint().to_dict()
+            elif isinstance(constraint, Constraint):
+                constraint = constraint.to_dict()
+
+            constraint = Constraint.from_dict(constraint)
+            self._constraints[idx] = constraint
+
+        self._constraints.sort(key=self._get_constraint_sort_key)
+
+    def _validate_constraint_order(self):
+        transform_constraints = list()
+        for constraint in self._constraints:
+            try:
+                if constraint.handling_strategy == 'transform':
+                    transform_constraints.append(constraint)
+            except AttributeError:
+                pass
+
+        for i in range(len(transform_constraints) - 1):
+            cur_constraint = transform_constraints[i]
+            for j in range(i + 1, len(transform_constraints)):
+                next_constraint = transform_constraints[j]
+                try:
+                    for column in cur_constraint.rebuild_columns:
+                        if column in next_constraint.constraint_columns:
+                            raise Exception('Multiple constraints will modify the same column: '
```

Comment:
I think this message could be more helpful if it told the user which constraints caused the issue. Also, I think there's a more efficient way to implement this without using nested for loops O(n^2) by using a counter?

Reply:
What do you mean by using a counter?